### PR TITLE
Add contribution link for current page

### LINKED ISSUE #13: Add a Contribute link
Add a link in the footer to GitHub in edition mode for the current page.

### PR DESCRIPTION
**Pull request by sandcha**
Fixes #13 

* Adds a contribution link on the homepage footer
* The contribution link is calculated: it's the current page path on GitHub in edition mode 

Limitations 
* Doesn't manage directories (all pages are in `public/`).
* As GitHub link is in edition mode, it shows the login page first when the user isn't logged.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -54,6 +54,17 @@
...
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
+				<li><a id="github">⚙ Contribute to this page</a></li>
+			</ul>
+		</footer>
+		<script type="text/javascript">
```
Comment (Anna-Livia) [inline public/index.html]: Is it necessary to have a script here when there is only one page ?

File: public/index.html [hunk 2]
```diff
@@ -54,6 +54,17 @@
 			</figure>
 		</section>
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
```
Comment (Anna-Livia) [inline public/index.html]: Should we have a link here knowing it's the same page ?

File: public/index.html [hunk 3]
```diff
@@ -54,6 +54,17 @@
...
+				<li><a id="github">⚙ Contribute to this page</a></li>
+			</ul>
+		</footer>
+		<script type="text/javascript">
+			const filename = window.location.pathname.split("/").pop()
+			const thisOnGithub = "https://github.com/openfisca/openfisca.org/edit/master/public/" + filename
+			document.getElementById('github').href = thisOnGithub
```
Comment (Anna-Livia) [inline public/index.html]: As href is the only attribute here, is it necessary to define the whole <a> in the script ? Could we define only the href string and insert it in a <a href=thisOnGithub> ?

File: public/assets/css/main.css [hunk 4]
```diff
@@ -78,6 +78,31 @@ figcaption {
...
+ul {
+	list-style-type: none;
+	font-size: 80%;
+}
+
+ul li {
+	margin: 1em;
+}
+
+footer {
```
Comment (Anna-Livia) [inline public/assets/css/main.css]: text-align: center; to center the text

File: public/assets/css/main.css [hunk 5]
```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
+	font-size: 80%;
+}
+
+ul li {
```
Comment (Anna-Livia) [inline public/assets/css/main.css]: display: inline; to have the links one after the other

File: public/assets/css/main.css [hunk 6]
```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
```
Comment (Anna-Livia) [inline public/assets/css/main.css]: Why not put this in footer ul ?

File: public/assets/css/main.css (hunk 5 above)
Comment (Anna-Livia) [inline public/assets/css/main.css]: Why not put this in footer li ?

File: public/index.html [hunk 7]
```diff
@@ -54,6 +54,17 @@
 			</figure>
 		</section>
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
+				<li><a id="github">⚙ Contribute to this page</a></li>
```
Comment (Anna-Livia) [inline public/index.html]: > I'm not found of the gear :/ ⚙

File: public/index.html (hunk 2 above)
Comment (sandcha) [reply public/index.html]: Link removed but item kept in order to display the same footer on every page.

File: public/index.html (hunk 7 above)
Comment (sandcha) [reply public/index.html]: Would a GitHub favicon be better?

File: public/index.html (hunk 1 above)
Comment (sandcha) [reply public/index.html]: Yes in order to extend this functionality to all pages easily at the end of this sprint.

File: public/assets/css/main.css (hunk 6 above)
Comment (sandcha) [reply public/assets/css/main.css]: The same footer is used for multiple pull requests. 😓 In #19 we have multiple lists and some aren't in the footer.

File: public/assets/css/main.css (hunk 5 above)
Comment (sandcha) [reply public/assets/css/main.css]: `display: inline;` didn't work but links set one after the other using `flex` (thus, similar css to the on sections).

Comment (sandcha) [reply public/assets/css/main.css]: Same reason for `li` and `ul`, multiple small pull requests but similar css (as described above).

File: public/assets/css/main.css (hunk 4 above)
Comment (sandcha) [reply public/assets/css/main.css]: Footer text set to `center` using other property.

File: public/assets/css/main.css [hunk 8]
```diff
@@ -101,6 +101,39 @@ figcaption {
...
+	border-width: 2px;
+	border-color: #FFF;
+
+	margin-top: 2em;
+	padding: 3em;
+
+	background-color: #222;
+}
+
+footer ul {
```
Comment (fpagnoux) [inline public/assets/css/main.css]: There are `footer ul` selector in this file

File: public/assets/css/main.css [hunk 9]
```diff
@@ -101,6 +101,39 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
```
Comment (fpagnoux) [inline public/assets/css/main.css]: This is also defined in `footer ul`. You can remove the `footer ul` one as, defining it for `ul` is enough

**Pull request merged**